controllers/admin/AdminAjaxPs_buybuttonliteController.php

```diff
@@ -92,7 +92,15 @@ public function getSqlQuery($query, $id_lang)
     public function getProductImage($id_product)
     {
         $product = new Product($id_product);
+
         $link_rewrite = $product->link_rewrite;
+        if (is_array($link_rewrite)) {
+            $i = 0;
+            while (empty($product->link_rewrite[$i])) {
+                $i++;
+            }
+            $link_rewrite = $product->link_rewrite[$i];
```

Comment:
I don't understand what you are trying to do here? getting the last element in link_rewrite?